corporate/lib/support.py

```diff
@@ -156,6 +157,15 @@ def get_current_plan_data_for_support_view(billing_session: BillingSession) -> P
         customer=customer,
         current_plan=plan,
     )
+
+    # A customer with or without a current plan can have a fixed_price next plan configured.
+    if customer and customer.required_plan_tier:
```

Comment:
This is fine for now, but presumably this will get reworked when we get around to replacing the existing discount feature on `required_plan_tier` with a different `CustomerPlanOffer` variant.